src/style-transformer.js

```diff
@@ -338,6 +338,8 @@ class StyleTransformer {
     if (isNth) {
       selector = this._twiddleNthPlus(selector);
     }
+    selector = selector.replace(DIR_PAREN, (m, before, dir, after) =>
+      `[dir="${dir}"] ${before}${after}, ${before}[dir="${dir}"]${after}`);
```

Comment:
Why should the "before" come after `[dir="${dir}"] `?

Reply:
If this is an important case, we need a test.

Reply:
To support things like `div:dir(rtl)`. The "before" portion would be `div`.